HW_5/task5_3.py

```diff
@@ -0,0 +1,13 @@
+# Создайте функцию генератор чисел Фибоначчи
+
+
+
+def fibonachi(n):
+    a = 0
+    b = 1
```

Comment:
Парная присваивание – маст хэв.
```suggestion
    a, b = 0, 1
```